src/main/java/bio/terra/cli/context/GlobalContext.java

```diff
@@ -170,43 +165,82 @@ public void updateDockerImageId(String dockerImageId) {
   //       - sub-directory for persisting pet SA keys: pet-keys/[terra user id]
   //           - pet SA key filename: [workspace id]
 
-  /** Getter for the global context directory. */
-  public static Path resolveGlobalContextDir() {
-    // TODO: allow overriding the global context directory path (e.g. env var?)
-    return DEFAULT_GLOBAL_CONTEXT_DIR;
+  /**
+   * Get the global context directory path relative to the user home directory.
+   *
+   * @return absolute path to global context directory, relative to the user home directory
+   */
+  @JsonIgnore
+  public static Path getGlobalContextDir() {
+    // TODO: allow overriding the global context directory path (e.g. env var != user home
+    // directory)
+    return getGlobalContextDir(Paths.get(System.getProperty("user.home")));
   }
 
   /**
-   * Getter for the sub-directory of the global context directory that holds the pet SA key files
-   * for all users.
+   * Get the global context directory path relative to the given directory.
```

Comment:
This Javadoc and the method signature are confusing. It would help to give an example or two, and possibly be more precise with expectations around relative vs. absolute paths on the input and output.

(That said, Will's suggestion to adjust the approach and use explicit overrides, rather than a generic "relativeToDir" param, will obviate most of the need for documentation here.)

Reply:
Replaced this method with logic in the DockerAppsRunner class, and included examples there.